### DISCUSSION TITLE
Creality Cr-6 Se and Max profiles

### DISCUSSION THREAD

**Discussion by bhast2** [OrcaSlicer/OrcaSlicer#1579] [Ideas]
I have made profiles for Creality's CR-6 SE and Max printers to work with OrcaSlicer. I have updated the files to github here is the link if you would like to include them in a future release.

https://github.com/bhast2/CR-6Se-Max-OrcaSlicer-profile

Thank you

**Comment by bhast2** [comment]
Is this a possibility to add this??

**Reply by Pumba763** [reply]
anychance you wanted to add the Creality CR-10 Smart and the CR-10 Smart Pro? (not the CR-10s that is a different printer again) 
they are similar to the CR-6 SE

**Reply by Pumba763** [reply]
And thanks for doing the 6se that was super helpful

**Comment by Oblix700** [comment]
Once installed.... I wanted to run calibration but it says add G92 E0 to layer_gcode


Please help

**Comment by afmc1** [comment]
I get this message on CR6SE!
![image](https://github.com/SoftFever/OrcaSlicer/assets/92735306/69566a87-f1a6-4b17-b733-f388826a0bd2)

**Comment by mryesman686** [comment]
can we move out the droop? I used to have without drooping under Machine G-code, however, i can't input it under orcaslicer. anyhelp? thanks

`M201 X500.00 Y500.00 Z100.00 E5000.00 ;Setup machine max acceleration
M203 X500.00 Y500.00 Z10.00 E50.00 ;Setup machine max feedrate
M204 P500.00 R1000.00 T500.00 ;Setup Print/Retract/Travel acceleration
M205 X8.00 Y8.00 Z0.40 E5.00 ;Setup Jerk
M220 S100 ;Reset Feedrate
M221 S100 ;Reset Flowrate

; Heat nozzle just enough to accurately home without drooping
M140 S50
M104 S120
M190 S50
M109 S120
M105


; - Home - this is used with the strain gauge as a zero position for the Z-offset
G28


; Now we know the positions we can move out of the way and happily droop at the side of the bed
M104 S[material_print_temperature_layer_0]
M140 S[material_bed_temperature_layer_0]

; Now we know the positions we can move out of the way and happily droop at the side of the bed
M104 S[material_print_temperature_layer_0]      ; You can also use material_print_temperature here
M140 S[material_bed_temperature_layer_0]  ; Or use material_bed_temperature
M105


G0 Z25
G0 X0 Y0

; Now wait for the temperatures to reach
M190 R[material_bed_temperature_layer_0]
M109 S[material_print_temperature_layer_0]

; And we can now draw our purge line
G92 E0 ;Reset Extruder
G1 Z2.0 F3000 ;Move Z Axis up
G1 X0 Y20 Z0.28 F5000.0 ;Move to start position
G1 X0 Y200.0 Z0.28 F1500.0 E15 ;Draw the first line
G1 X0.3 Y200.0 Z0.28 F5000.0 ;Move to side a little
G1 X0.3 Y20 Z0.28 F1500.0 E30 ;Draw the second line

G92 E0 ; Reset Extruder
G1 Z4.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed`
